.github/workflows/application.yml

```diff
@@ -170,8 +170,7 @@ jobs:
       artifacts_path: application/account-management/Api/publish
       image_name: account-management
       version: ${{ needs.build-and-test.outputs.version }}
-      docker_context: ./application/account-management
-      docker_file: ./Api/Dockerfile
+      api_location: ./application/account-management/Api/Api.csproj
```

Comment:
Should/could this be relative like the `artifacts_path`? So `application/account-management/Api/Api.csproj` without the `./`.